### PR TITLE
Fluid/feature analysis tests

### PR DESCRIPTION
**Pull request by jcotela**
Adding a test for the new analysis script. The plan is to eventually add more (or migrate a few of the existing ones), but I figure that one test is still better than no tests, for now.

### CODE REVIEW

File: applications/FluidDynamicsApplication/tests/fluid_analysis_test.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        self.print_output = False
+
+    def testFluidDynamicsAnalysis(self):
+        work_folder = "CylinderTest"
+        settings_file_name = "cylinder_fluid_parameters.json"
+
+        with WorkFolderScope(work_folder):
+            self._run_test(settings_file_name)
+
+            self._delete_file_if_existing("cylinder_2d.time")
```
Comment (philbucher) [inline applications/FluidDynamicsApplication/tests/fluid_analysis_test.py]: this exists in the core in kratos_utilities

**Review by philbucher — DISMISSED** [review]
Nice!

It is too slow for the SmallSuite?

File: applications/FluidDynamicsApplication/tests/fluid_analysis_test.py (hunk 1 above)
Comment (jcotela) [reply applications/FluidDynamicsApplication/tests/fluid_analysis_test.py]: I did not know... I'll use the standard one then.

**Comment by jcotela** [comment]
@philbucher yes, it turned out to be too slow for the small queue, since it is a relatively big case.

**Review by philbucher — APPROVED** [review]

**Pull request merged**